### ISSUE TITLE
[Request] Sakura Wars GB [en patch]

### ISSUE THREAD

**Issue by BOBdotEXE** [mattcurrie/gb-save-states#30]
Not sure if it's possible on larger games like this,
but  Sakura Wars GB is full of RNG and has lots of missable stuff,
So save states would be great!

https://www.romhacking.net/translations/6348/

**Comment by mattcurrie** [comment]
Hi there,

I've now created a patch for this. Give it a try and let me know if any issues :) 

Note I had to fix a bug in the game during the introduction which caused it to incorrectly write to the SRAM bank change register. This wouldn't have been an issue with the original game because there was only one SRAM bank, but now with multiple SRAM banks (for the save states patch) this bug was causing an issue. Anyway hopefully that was the only such bug in the game.
